mmcv/runner/optimizer/default_constructor.py

```diff
@@ -115,23 +143,35 @@ def add_params(self, params, module, prefix=''):
                 warnings.warn(f'{prefix} is duplicate. It is skipped since '
                               f'bypass_duplicate={bypass_duplicate}')
                 continue
-            # bias_lr_mult affects all bias parameters except for norm.bias
-            if name == 'bias' and not is_norm:
-                param_group['lr'] = self.base_lr * bias_lr_mult
-            # apply weight decay policies
-            if self.base_wd is not None:
-                # norm decay
-                if is_norm:
-                    param_group[
-                        'weight_decay'] = self.base_wd * norm_decay_mult
-                # depth-wise conv
-                elif is_dwconv:
-                    param_group[
-                        'weight_decay'] = self.base_wd * dwconv_decay_mult
-                # bias lr and decay
-                elif name == 'bias':
+            # if the parameter match one of the custom keys, ignore other rules
+            is_custom = False
+            for key in custom_keys:
```

Comment:
We may sort the `custome_keys` according to its length. 
In this way, the most specific key will be matched first. In other words, `priority = len(key)`. 
In this way, when we have some like 
```python
custom_keys={
       'backbone.conv1': {'lr_mlt': 1.},
       'backbone': {'lt_mult':10}
```
we could set `backbone` `lr_mult` w.o. changing `backbone.conv1` `lr_mult`.

Reply:
You can take a look at the docstring~